pkg/catalog/ingress.go

```diff
@@ -43,8 +44,15 @@ func (mc *MeshCatalog) GetIngressTrafficPolicy(svc service.MeshService) (*traffi
 			continue
 		}
 
+		if backend.Port.Number < 0 {
+			// to prevent panics since (until this commit) there was no validating webhook to check
+			backend.Port.Number = 0
+		}
```

Comment:
Where does this panic?
The allowed port number range is a part of the CRD schema.

Reply:
Ah, didn't notice that; I'll remove the check then